Firmware/Marlin.h

```diff
@@ -22,6 +22,7 @@
 #include "pins.h"
 #include "Timer.h"
 #include "mmu2.h"
+#include "cardreader.h" // for IS_SD_PRINTING
```

Comment:
Do we need to move this here? I know it's more convenient, but I'd try to keep includes in Marlin.h as low as possible as this is included everywhere, resulting in total rebuilds if you touch any of the dependencies.

Reply:
Yes, I would otherwise get a compiler error. `IS_SD_PRINTING` is used in `Marlin.h` but it was being included from `mmu2.h` before.

Reply:
![image](https://user-images.githubusercontent.com/8218499/208297802-4f483fb8-32c5-4a4a-845f-c743bfd11a2e.png)

Reply:
Found an alternative. Making CHECK_FSENSOR into a function. 👍

Reply:
Costs no additional memory :D